### ISSUE TITLE
bug: add_aromatic_interactions get None DataFrame

### ISSUE THREAD

**Issue by EyalRozenberg1** [a-r-j/graphein#81]
In `graphein/protein/edges/distance.py`, function `add_aromatic_interactions`. The `pdb_df` dataframe is None by default. The following lines should be probably added (at line 261):
```
    if pdb_df is None:
        pdb_df = G.graph["pdb_df"]
```

**Comment by johnnytam100** [comment]
Hi, I'm sorry for reporting several situations at the same time. I met the same problem using the `add_aromatic_interactions`

```
from graphein.protein.edges.distance import (add_peptide_bonds,
                                             add_hydrogen_bond_interactions,
                                             add_disulfide_interactions,
                                             add_ionic_interactions,
                                             add_aromatic_interactions,
                                             add_aromatic_sulphur_interactions,
                                             add_cation_pi_interactions
                                            )

new_edge_funcs = {"edge_construction_functions": [add_peptide_bonds,
                                                  add_hydrogen_bond_interactions,
                                                  add_disulfide_interactions,
                                                  add_ionic_interactions,
                                                  add_aromatic_interactions,
                                                  add_aromatic_sulphur_interactions,
                                                  add_cation_pi_interactions
                                                  ]
                 }

config = ProteinGraphConfig(**new_edge_funcs)
g = construct_graph(config=config, pdb_code="3ED8")
p = plot_protein_structure_graph(G=g, angle=0, colour_edges_by="kind", colour_nodes_by="seq_position", label_node_ids=False)
plt.suptitle("Protein graph with: peptide backbone, H-Bonds, \n Disulphide, ionic, aromatic, aromatic-sulphur and cation-pi interactions. \n  Nodes coloured by sequence position, edges by type")

import pandas as pd
print(pd.__version__)
```

May I know is it the problem with `pandas` version?

**Comment by a-r-j** [comment]
Hey @johnnytam100 this should be fixed in graphein 1.1.0 (`pip install graphein==1.1.0`)

[Colab example](https://colab.research.google.com/drive/17tbbY08ERiy7pwtVrHbiYS-2_clr_oBV?usp=sharing) 

Looks like I'd made the fix & not updated the package 🙄 Sorry!

Closing this now, do let me know if it's still not working!
